### PR TITLE
Updated: mysql-client to mariadb-client

### PR DESCRIPTION
**Pull request by GoliathLabs**
Updated mysql-client to mariadb-client in the hope that the action doesn't fail again

### CODE REVIEW

**Comment by GoliathLabs** [comment]
Looks like everything is working again :) @rixx

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -18,17 +18,8 @@ jobs:
     name: Tests
     strategy:
       matrix:
-        python-version: ['3.8', '3.9', '3.10']
+        python-version: ["3.9", "3.10"]
```
Comment (rixx) [inline .github/workflows/tests.yml:21]: We should keep Python 3.8 in the array of supported/tested versions, as it's still supported and common, and there's no real reason to remove it, as far as I can see.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -18,17 +18,8 @@ jobs:
...
         database: [sqlite, postgres, mysql]
-        exclude:
```
Comment (rixx) [inline .github/workflows/tests.yml:23]: At the same time, excluding some test combinations makes testing faster and more likely to be actually performed in full. Is there a reason you removed these excludes?

**Comment by GoliathLabs** [comment]
Readded them back. You're right, that some tests may be excluded. Should be ready for the merger now

**Review by rixx — APPROVED** [review]
Thanks!

**Pull request merged**